src/utils/components/NetworkInterfaceModal/components/NetworkInterfaceNetworkSelect.tsx

```diff
@@ -111,37 +128,72 @@ const NetworkInterfaceNetworkSelect: FC<NetworkInterfaceNetworkSelectProps> = ({
   ]);
 
   return (
-    <FormGroup fieldId="network-attachment-definition" isRequired label={t('Network')}>
+    <FormGroup
+      labelIcon={
+        <Popover
+          bodyContent={
+            <Trans t={t}>
+              <div>
+                <b>Bridge binding</b>: Connects the VirtualMachine to the selected network, which is
+                ideal for L2 devices.
+              </div>
+              <div className="NetworkInterfaceNetworkSelect__popover">
+                <b>SR-IOV binding</b>: Attaches a virtual function network device to the
+                VirtualMachine for high performance.
+              </div>
+              <ExternalLink
+                href="https://docs.openshift.com/container-platform/4.15/virt/vm_networking/virt-networking-overview.html#secondary-network-config"
+                text={t('Learn more')}
+              />
```

Comment:
we can remove the `t` function or move it outside the `Trans`

Reply:
I've LGTMed the PR and then I saw this. Yes, let's remove the `t` function and then we can remove the "hold" label :)

Reply:
Moved outside of Trans component. Thanks :)